### ISSUE TITLE
Updated Ubuntu Version

### ISSUE THREAD

**Issue by jbrumwell** [pascalewalters/homography_annotation_docs#6]
@pascalewalters curious if you plan on releasing an updated ubuntu 18 version with the new features of the windows version? I would love to give guidelines a try.

Secondly it would be awesome if we could have a mask generated similar to this;

![001_AB](https://user-images.githubusercontent.com/413363/61332446-371bbe00-a7f2-11e9-9eed-2796ceabf0b2.jpg)

What do you think?

Thank you @pascalewalters :)

**Comment by pascalewalters** [comment]
Hi @jbrumwell,

I'll work on updating the existing Ubuntu version to add the new features. 

I haven't really done any work in terms of soccer field segmentation, since this project is mostly for hockey. You would be welcome to try to make your own, using the hockey model as a template. Let me know how it goes :)

**Comment by jbrumwell** [comment]
@pascalewalters thank you for sharing your work it is great :) The above image is from a dataset related to soccer, I as well am interested in only the hockey model. Was curious if the right side of the image could be generated from the tool.

**Comment by pascalewalters** [comment]
Your aim would be to segment the ice surface without the lines included, right? I can add a mask for that

**Comment by jbrumwell** [comment]
Yes that is correct, I'm looking to generate a dataset for https://github.com/lood339/pytorch-two-GAN the goal is player localization on the ice.

**Comment by jbrumwell** [comment]
@pascalewalters got it up and running in wine so don't rush on the Ubuntu version for my sake :+1: Thanks again :)
